backend/src/main/java/com/crm/model/db/CarEntity.java

```diff
@@ -0,0 +1,23 @@
+package com.crm.model.db;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+
+@Entity(name = "car_entity")
+public class CarEntity {
+
+    @Id
+    private int id;
+
+    private String vin;
+    private String registrationNumber;
+    private String brand;
+    private String model;
+    private int productionYear;
+    private int mileage;
+
+    @ManyToOne(cascade = CascadeType.ALL)
+    private CarType carType;
```

Comment:
Should be named CarTypeEntity to be in line with adopted naming of entities. Or change CarEntity to Car and leave CarType for this.

Reply:
done